wp-content/themes/core/components/blocks/interstitial/Interstitial_Block_Controller.php

```diff
@@ -142,6 +147,19 @@ public function get_content_args(): array {
 		];
 	}
 
+	/**
+	 * @return \Tribe\Project\Templates\Components\Deferred_Component
+	 */
+	private function get_leadin(): Deferred_Component {
+		return defer_template_part( 'components/text/text', null, [
+			Text_Controller::CLASSES => [
+				'c-block__leadin',
+				'b-tabs__leadin',
```

Comment:
I think you can nix this. This looks like a class specific to the tabs block

Reply:
Good catch, thank you!